### ISSUE TITLE
test: Multiple deployment: Multiple deployment of the stack in same account (using different input prefix)

### ISSUE THREAD

**Issue by vburckhardt** [terraform-ibm-modules/stack-retrieval-augmented-generation#32]
### Description

Context: https://github.com/terraform-ibm-modules/stack-retrieval-augmented-generation/issues/17

**Comment by rajatagarwal-ibm** [comment]
when I created multiple deployments (stacks) in the same project with different prefixes, it started to deploy the already deployed stack again.

Steps I followed:

- Created a stack in the project deployment1
- Deployed deployment1 successfully.
- Created another stack in the project deployment2
- Ran the script to deploy the deployment2 stack

Observations -

- Changed status of deployment1 stack from "deployed" to "waiting"
- deployment2 stack looks paused. I assume it will deploy after the deployment1 stack re-deployed. The execution is currently going on, I will confirm this hypothesis once the execution is completed.
